packages/vuetify/src/components/VSlider/__tests__/VSlider.spec.ts

```diff
@@ -294,14 +294,17 @@ describe('VSlider.ts', () => {
     expect(wrapper.vm.thumbPressed).toBe(false)
     expect(wrapper.vm.isActive).toBe(false)
 
-    container.trigger('mousedown', {
-      touches: [],
+    container.trigger('touchstart', {
+      touches: [{}],
     })
 
     await wrapper.vm.$nextTick()
 
-    expect(wrapper.vm.thumbPressed).toBe(true)
```

Comment:
Why is this not set anymore?

Reply:
Thanks for review. I have left out it and will add it soon.

Reply:
done~